generators/app/index.js

```diff
@@ -166,13 +175,14 @@ module.exports = class extends Generator {
         oSubGen.modulename = "uimodule";
 
         if (oConfig.platform !== "Static webserver" && oConfig.platform !== "SAP NetWeaver") {
-            this.composeWith(require.resolve("../additionalmodules"), oSubGen);
+            this.composeWith(__dirname + "/../additionalmodules", oSubGen);
         }
 
-        this.composeWith(require.resolve("../newwebapp"), oSubGen);
+        this.composeWith(__dirname + "/../newwebapp", oSubGen);
```

Comment:
Does this work on a windows machine? (I don't have one, otherwise, I would try ;) ). I'd expect that you need to use something like `path.join(__dirname, "/../additionalmodules")`

Reply:
Good point, I don't know. Can someone try? 😀

Reply:
+1 on tobias feedback: please make it cross-OS compatible by composing via `path`